### ISSUE TITLE
Indent error after formatting a template string

### ISSUE THREAD

**Issue by chrishan1995** [prettier/prettier#9628]
**Prettier 2.1.2**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEEBGArOMLAXk2AB0pNMByeDAQRqUwANLrqAeACwCYAfAGUeEAA5iAllADmmAGIQATpgAqAQxwAbOJgAy6gLZi0XAPT8BHTlzECAogEcArpLGYAnhGcqAJnDQAa2wVLnVMHiU4ADNichAzNBkAEmAZOBgAVSUtAAoaHDhfNEl-AFoYTR00GgBKAF94gULi0t1K7ThzdQFMAHdJGB5MId0xOCVowiwOnUwtIzEAOkwAYSj1eBGeXTb1LUwMZ18PQ7F1MF0BocxwnCVJGR4Zqt0F423NzDB1Kh2tdxeZyYaIQMDONAAGhChxgHjmQNC4UiMTiCSSqXSWRy+RaJXKswCdUaIGaRXx7Ve3V612G4UJ80W-UGwzQEGiMGhUhkhgQ8F8I2gARBylu3wgAC9ThtfNI5JAlFAJtgqIjMC0VgAhB6yMXjJRsqD7QXOMDDGAQW5QaUQCCGZk3bwqBnvExLcx2aysSHWOgBGCa5i0egwGg+qD1ECQkDiGCSdDIUDqJRKCB9AAKyYQaGQIH2fXUHhz0fuF0CGSE5zAcuQMCUzjg0bghha-l8BlkznU6UUSkMmzjslz6mcFqjIGehi0AHUeIMAlW4EJs4NJAA3QYeXNgNDFkDSNATGDppTd-vIaL7Q-RvBoAAe2rLFaMcD00jgF6vjZAt7vQjlOgAIrOBA8C1vW37nAaEy5jgmhwFo45iDqMDTqUQzIAAjHwAAM0bIRAh7TqeYi5shAQTGuH7Ri4oFwCe4g5ig6hoGUSpFEU45RC4khRCeZ7qJ+WjXiAh6GJI4ENtGJSyEBIFgUgdZSSAHRob4GFIHw0Z1uokhaHKqx2ueKABAArOOEJwBoOBMZewnfmuDYAJJQP4sBCGADxiDADCuUIcI6EJh71PUQA)
```sh
--jsx-single-quote
--parser babel
--print-width 120
--single-quote
```

**Input:**
```jsx
const object = {
  'testA': `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  'testB': 'test',
}
```

**Output:**
```jsx
const object = {
  testA: `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl(
      'bedside-tables'
    )}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl(
    'bedside-tables'
  )}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  testB: 'test',
};

```

**Expected behavior:**
```jsx
const object = {
  'testA': `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  'testB': 'test',
}
```

or 

```jsx
const object = {
  testA: `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl(
      'bedside-tables'
    )}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl(
      'bedside-tables'
    )}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  testB: 'test',
};

```

**Comment by rattrayalex** [comment]
I'm fairly certain this is intended behavior, though I can see an argument for why it isn't desirable.

**Comment by fisker** [comment]
Fixed by #13621

**Prettier pr-9583**
[Playground link](https://deploy-preview-9583--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuc0DOMAEEBGArOMLAXk2AB0pNMByeDAQRqUwANLrqAeACwCYAfAGUeEAA5iAllADmmAGIQATpgAqAQxwAbOJgAy6gLZi0XAPT8BHTlzECAogEcArpLGYAnhGcqAJnDQAa2wVLnVMHiU4ADNichAzNBkAEmAZOBgAVSUtAAoaHDhfNEl-AFoYTR00GgBKAF94gULi0t1K7ThzdQFMAHdJGB5MId0xOCVowiwOnUwtIzEAOkwAYSj1eBGeXTb1LUwMZ18PQ7F1MF0BocxwnCVJGR4Zqt0F423NzDB1Kh2tdxeZyYaIQMDONAAGhChxgHjmQNC4UiMTiCSSqXSWRy+RaJXKswCdUaIGaRXx7Ve3V612G4UJ80W-UGwzQEGiMGhUhkhgQ8F8I2gARBylu3wgAC9ThtfNI5JAlFAJtgqIjMC0VgAhB6yMXjJRsqD7QXOMDDGAQW5QaUQCCGZk3bwqBnvExLcx2aysSHWOgBGCa5i0egwGg+qD1ECQkDiGCSdDIUDqJRKCB9AAKyYQaGQIH2fXUHhz0fuF0CGSERjgemkcGQ0X2aDgJaUZYr5zAcuQMCUzmbIDghha-l8BlkznU6UUSkMmzjslz6mcFqjIGehi0AHUeIMAh24EJs4NJAA3QYeXNgNDFkDSJtKGDp1s89T1xv9vBoAAeQjlOgAis4EDwG+WhNtG5wGhMuY4JocBaKuYg6jAm6lEMyAABwAAwQamTabq2Yi5khAQTCedbRi4wFwE+4g5ig6hoGUSpFEUq5RC4khRE+k6zqB4EgE2hiSN2vb9iUsgAUBIFIA2YH9h0qG+OhSB8NGPbqJIWhyqsdp8QxWgIdGEJwBoOD0XJAknn2ACSUD+LAQhgA8YgwAw9lCHCOj8XA9T1EAA)

```sh
--parser babel
```

**Input:**

```jsx
const object = {
  'testA': `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl('bedside-tables')}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  'testB': 'test',
}
```

**Output:**

```jsx
const object = {
  testA: `
    <h2>Shopping For Table Lamps</h2>
    <p>Equip your desk or <a href="/sg${getUrl(
      "bedside-tables",
    )}">bedside table</a> with the perfect table lamp. Create the ideal study space with a bright table lamp that can help you focus, or style your <a href="/sg${getUrl(
      "bedside-tables",
    )}">bedside table</a> with a table lamp with soft, pigmented tones for a cozy reading corner on your bed. Bring a personal touch to any room with our table lamps.</p>
  `,
  testB: "test",
};

```
